src/current/cockroachcloud/plan-your-cluster-serverless.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Plan a CockroachDB Serverless Cluster
+summary: Plan your cluster's configuration.
+toc: true
+docs_area: deploy
+cloud: true
+---
+
+{% include cockroachcloud/filter-tabs/plan-your-cluster.md %}
+
+This page describes how resource usage, pricing, and cluster configurations work in {{ site.data.products.serverless }}.
+
+## Request Units
+
+All cluster activity, including SQL queries, bulk operations, and background jobs, is measured in Request Units, or RUs. RUs are an abstracted metric that represent the size and complexity of requests made to your cluster. [Pricing](https://cockroachlabs.com/pricing) shows cost estimates of some common queries and how they increase with the size and complexity of the query. In addition to queries that you run, Request Units can be consumed by background activity, such as automatic statistics gathering used to optimize your queries or changefeeds connected to an external sink.
```

Comment:
> [Pricing](https://cockroachlabs.com/pricing) shows cost estimates of some common queries
Wasn't this page deleted as part of this PR? What would this link to?

Reply:
this link is to the marketing page on the main site, but it looks like they didn't add the query table back in there so i'm going to delete it

Reply:
Adding RU estimates back in without the prices

Reply:
Reverting to original link out to pricing page now that marketing is publishing these again